lib/index.js

```diff
@@ -116,48 +116,6 @@ var nameSpaceIds = {
     CategoryTalk: 15,
 };
 nameSpaceIds.Image = nameSpaceIds.File;
-
-
-/**
- * Checks whether namespace is `Media`
- *
- * @method
- * @name Namespace.prototype#isMedia
- * @return {boolean}
- */
-
-/**
- * Checks whether namespace is `Special`
- *
- * @method
- * @name Namespace.prototype#isSpecial
- * @return {boolean}
- */
-
-/**
- * Checks whether namespace is `Main`
- *
- * @method
- * @name Namespace.prototype#isMain
- * @return {boolean}
- */
-
-/**
- * Checks whether namespace is `Talk`
- *
- * @method
- * @name Namespace.prototype#isTalk
- * @return {boolean}
- */
-
-/**
- * Checks whether namespace is `UserTalk`
- *
- * @method
- * @name Namespace.prototype#isUserTalk
- * @return {boolean}
- */
-
```

Comment:
Docs in `README.md` are generated from these comments using the [jsdoc-to-markdown](https://www.npmjs.com/package/jsdoc-to-markdown) util. I think I need to add an npm script to generate API docs and run it as a pre-commit hook. So I think we shouldn't remove these comments.